AjobK/Athena
# 97 migrations

closes #97 

 **What is it supposed to do**
This is a database branch where DB files were added such like more migrations in order to update the database itself or ERD files

## PLEASE READ AGAIN

 **How can we test your branch**
for the migration

First we have to delete the current database to make everything easier for ourselves
1. First navigate to XAMPP
2. Select the DB 'seaqull'
3. Go to 'Actions' or 'Handelingen' up top 
![image](https://user-images.githubusercontent.com/42764788/59068553-b8f3ff80-88b4-11e9-8d52-2c1c4fd32aa5.png)
4. Then press the 'delete database' text, it can't be missed. It's highlighted in red
![image](https://user-images.githubusercontent.com/42764788/59068633-f062ac00-88b4-11e9-8507-02545743bdc2.png)
5. Confirm
6. Create a new DB within XAMPP called 'seaqull'
7. Done :D

Now let's fill it up
1. Navigate to seaqull/passport (`cd passport`)
2. Do `php artisan migrate`
3. Go to phpMyAdmin and view the DB changes
4. Run `php artisan migrate:rollback` (in the case that this did not roll back all migrations, run `migrate:reset instead`
5. Run step 2 again
6. Let me know if anything fails

**How can we view the ERD**
Just take a look at the ERD files in `seaqull/src/database`
